user_docs/en/userGuide.t2t

```diff
@@ -1447,8 +1447,14 @@ NVDA will ask if you really wish to do this.
 As with installing, NVDA must be restarted for the add-on to be fully  removed.
 Until you do, a status of "remove" will be shown for that add-on in the list.
 
+To enable or disable an add-on, select an add-on from the list and press Enable or Disable button.
+You can disable an add-on if add-on status indicates it is running or enabled, or enable it if add-on is suspended or disabled.
+For each press of enable/disable button, add-on status changes to indicate what will happen when NVDA restarts.
+If you do choose to disable an add-on, be sure to confirm your choice when prompted.

```

Comment:
Per the suggestion that the warning when disabling is removed, this line can be also removed.